tools/bcz.cmd

```diff
@@ -68,7 +68,7 @@ if "%_EXCLUSIVE%" == "1" (
 echo Performing nuget restore...
 nuget.exe restore %OPENCON%\OpenConsole.sln
 
-set _BUILD_CMDLINE="%MSBUILD%" %OPENCON%\OpenConsole.sln /t:%_MSBUILD_TARGET% /m /p:Configuration=%_LAST_BUILD_CONF% /p:Platform=%ARCH% %_APPX_ARGS%
+set _BUILD_CMDLINE="%MSBUILD%" %OPENCON%\OpenConsole.sln /t:"%_MSBUILD_TARGET%" /m /p:Configuration=%_LAST_BUILD_CONF% /p:Platform=%ARCH% %_APPX_ARGS%

```

Comment:
wat? why?

Reply:
The _MSBUILD_TARGET can have spaces in it, and if it does, this won't work.Case in point: any project in the "Virtual Terminal" solution directory. Wrapping it with quotes fixes that.